compass/landice/tests/ismip6_forcing/ocean_basal/process_basal_melt.py

```diff
@@ -0,0 +1,232 @@
+
+            # place the output file in appropriate directory
+            if output_base_path == "NotAvailable":
+                return
+
+            output_path = f'{output_base_path}/basal_melt/parametrizations/'
+            if not os.path.exists(output_path):
+                print("Creating a new directory for the output data")
+                os.makedirs(output_path)
+
+            src = os.path.join(os.getcwd(), output_file)
+            dst = os.path.join(output_path, output_file)
+            shutil.copy(src, dst)
+
+            print("")
+
+    def combine_ismip6_inputfiles(self, basin_file, coeff_gamma0_deltaT_file,
+                                  combined_file_temp):
+        """
+        Combine ismip6 input files before regridding onto mali mesh
+
+        Parameters
+        ----------
+        basin_file : str
+            imbie2 basin numbers in ismip6 grid
+        coeff_gamma0_deltaT_file : str
+            uniform melt rate coefficient (gamma0) and temperature
+            correction per basin
+        combined_file_temp : str
+            temporary output file that has all the variables combined
+        """
+
+        ds_basin = xr.open_dataset(basin_file, engine="netcdf4")
+        ds = xr.open_dataset(coeff_gamma0_deltaT_file, engine="netcdf4")
+
+        ds["ismip6shelfMelt_basin"] = ds_basin.basinNumber
+        write_netcdf(ds, combined_file_temp)
+
+    def remap_ismip6BasalMelt_to_mali(self, input_file, output_file,
+                                      mali_mesh_name, mali_mesh_file,
+                                      method_remap):
+        """
+        Remap the input ismip6 basal melt data onto mali mesh
+
+        Parameters
+        ----------
+        input_file: str
+            temporary output file that has all the variables combined
+            combined_file_temp generated in the above function
+        output_file : str
+            ismip6 data remapped on mali mesh
+        mali_mesh_name : str
+            name of the mali mesh used to name mapping files
+        mali_mesh_file : str, optional
+            The MALI mesh file if mapping file does not exist
+        method_remap : str, optional
+            Remapping method used in building a mapping file
+        """
+
+        # check if mapfile exists
+        mapping_file = f"map_ismip6_8km_to_{mali_mesh_name}_{method_remap}.nc"
+
+        if not os.path.exists(mapping_file):
+            # build a mapping file if it doesn't already exist
+            build_mapping_file(self.config, self.cores, self.logger,
+                               input_file, mapping_file, mali_mesh_file,
+                               method_remap)
+        else:
+            print("Mapping file exists. Remapping the input data...")
+
+        # remap the input data
+        args = ["ncremap",
+                "-i", input_file,
+                "-o", output_file,
+                "-m", mapping_file]
+
+        subprocess.check_call(args)
```

Comment:
Again, this needs to use `mpas_tools.logging.check_call` and pass `self.logger`.  Otherwise, output goes to the terminal instead of the log file.